hypha/apply/dashboard/views.py

```diff
@@ -126,6 +130,15 @@ def projects_to_approve(self):
         }
 
     def my_reviewed(self, submissions):
+        '''
+        If use_settings variable is set for ReviewerSettings use settings
+        parameters to filter submissions or return only reviewed_by as it
+        was by default.
+        '''
+        reviewer_settings = ReviewerSettings.for_request(self.request)
+        if reviewer_settings.use_settings:
+            submissions = submissions.for_reviewer_settings(self.request.user, reviewer_settings)
```

Comment:
@frjo, this is AdminDashboardView this should be done only for [ReviewerDashboardView](https://github.com/HyphaApp/hypha/blob/4b865bb04106ab7a409fdf0dc3d09b75e7602c45/hypha/apply/dashboard/views.py#L175)

Reply:
Oops, fixed now.

Reply:
But you think the method is ok otherwise.

Reply:
Yeah. This looks good. I'm just wondering if this should also be done for [awaiting_reviews](https://github.com/HyphaApp/hypha/blob/4b865bb04106ab7a409fdf0dc3d09b75e7602c45/hypha/apply/dashboard/views.py#L201) and my_flagged.

Reply:
Good point!